docker/install-viral-ngs.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+mkdir -p /opt/viral-ngs/viral-ngs-etc
+ln -s /opt/viral-ngs/source /opt/viral-ngs/viral-ngs-etc/viral-ngs
+ln -s /opt/miniconda /opt/viral-ngs/viral-ngs-etc/conda-env
+export VIRAL_CONDA_ENV_PATH=/opt/miniconda
+mv /opt/docker/easy-deploy-viral-ngs.sh /opt/viral-ngs
+mv /opt/docker/env_wrapper.sh /opt/viral-ngs
```

Comment:
Why not copy the env_wrapper.sh to `/opt/viral-ngs` during the docker build since the copy is in the derivative image anyway and not the more generic base image?

Reply:
Good point, all of this stuff can/should go straight in.